requirements.txt

```diff
@@ -0,0 +1,7 @@
+chaostoolkit-lib>=0.18.0
+chaostoolkit>=0.10.0
+click
+logzero
+requests
+simplejson
```

Comment:
I wonder if the hub needs to import simplejson and yaml here since they are not directly used (only by the chaoslib internally IIRC).

Reply:
It seems to. Removal makes things fail.